File: src/main/java/net/ccbluex/liquidbounce/ui/client/clickgui/style/styles/yzygui/utils/Pair.java
/*
 * FDPClient Hacked Client
 * A free open source mixin-based injection hacked client for Minecraft using Minecraft Forge by LiquidBounce.
 * https://github.com/SkidderMC/FDPClient/
 */
package net.ccbluex.liquidbounce.ui.client.clickgui.style.styles.yzygui.utils;

public final class Pair<K, V> {

    private K key;
    private V value;

    public Pair(final K key, final V value) {
        this.key = key;
        this.value = value;
    }

    public void setKey(K key) {
        this.key = key;
    }

    public K getKey() {
        return key;
    }

    public void setValue(V value) {
        this.value = value;
    }

    public V getValue() {
        return value;
    }

}

File: src/main/java/net/ccbluex/liquidbounce/utils/render/Pair.kt
/*
 * FDPClient Hacked Client
 * A free open source mixin-based injection hacked client for Minecraft using Minecraft Forge.
 * https://github.com/SkidderMC/FDPClient/
 */
package net.ccbluex.liquidbounce.utils.render

data class Pair<K, V>(
    private var _key: K,
    private var _value: V
) {
    var key: K
        get() = _key
        set(value) {
            _key = value
        }

    var value: V
        get() = _value
        set(value) {
            _value = value
        }
}